src/libcrun/container.c

```diff
@@ -537,6 +537,30 @@ make_container (runtime_spec_schema_config_schema *container_def, const char *pa
   return container;
 }
 
+runtime_spec_schema_config_schema_process_user *
```

Comment:
I wrote this by hand because it seems like the YAJL-generated code doesn't include a deep-copy implementation for each generated struct, but please do let me know if there's a better way of doing this :slightly_smiling_face:

Reply:
Hi thanks, Could you please elaborate about what do you mean by this can go out of sync with generated code ? Did you mean that fields of `runtime_spec_schema_config_schema_process_user` might change ? 

I think maybe this function can be generated using `libocispec` and kept in a generated `C` file but I don't see why would that be necessity, wouldn't just `crun` not compile if following fields are changed and in that case developer can fix.

@giuseppe can give better insights.